main.go

```diff
@@ -37,6 +37,7 @@ var Options struct {
 	InsecureSkipVerify    bool   `envconfig:"INSECURE_SKIP_VERIFY" default:"false"`
 	ImageServiceBaseURL   string `envconfig:"IMAGE_SERVICE_BASE_URL"`
 	LogLevel              string `envconfig:"LOGLEVEL" default:"info"`
+	AdditionalCAFile      string `envconfig:"ADDITIONAL_CA_FILE" default:""`
```

Comment:
This name is not great, we already have `HTTPS_CA_FILE` a few lines above. Please choose a different name and document the difference between them